.github/workflows/ci.yml

```diff
@@ -209,24 +187,19 @@ jobs:
     - name: Download the packages from the Artifacts
       uses: actions/download-artifact@v4
       with:
-        name: 'Appx Packages (${{ env.CONFIGURATION }}, ${{ env.AUTOMATED_TESTS_ARCHITECTURE }})'
+        name: 'Appx Packages (${{ env.CONFIGURATION }}, ${{ env.PLATFORM }})'
         path: ${{ env.ARTIFACTS_STAGING_DIR }}
 
-    - name: Install Files
+    - name: Prepare for the tests
       shell: powershell
       run: |
         Set-Location "$env:APPX_PACKAGE_DIR"
         $AppxPackageBundleDir = Get-ChildItem -Filter Files.Package_*_Test -Name
         Set-Location $AppxPackageBundleDir
         ./Install.ps1 -Force
         Get-AppxPackage
-
-    - name: Set full HD resolution
-      run: Set-DisplayResolution -Width 1920 -Height 1080 -Force
-
-    - name: Start WinAppDriver process
-      shell: pwsh
-      run: Start-Process -FilePath "$env:WINAPPDRIVER_EXE86_PATH"
+        Set-DisplayResolution -Width 1920 -Height 1080 -Force
+        Start-Process -FilePath "$env:WINAPPDRIVER_EXE86_PATH"
```

Comment:
Why did you merge these into one step?

Reply:
Because I thought they hardly would fail and merging them would not harm us, rather, it contributes to having less lines of code.